### ISSUE TITLE
Bin doesn't work

### ISSUE THREAD

**Issue by Pitel** [sebascontre/bootscreen3ds#5]
I had to use https://xem.github.io/3DShomebrew/tools/image-to-bin.html to convert png to bin.

**Comment by sebascontre** [comment]
Can you provide more information about the problem? What browser you use? Online or Offline version? You see some type of error or something? The code we use to the bin converter is the same of the page you link it.

**Comment by Pitel** [comment]
Problem solved. I'm using AdBlocker (if you want beer/tea, send me your BTC address), and it blocked estatico.pw requests and for some reason, it broke whole bin generation (the bin file was full of zeros). Weird thing is png was ok.

**Comment by malicious50** [comment]
I have this problem now too. I used the splash before and worked like a charm, now I updated Luma to 7.0.1 and splash.bin file doesn't want to work. Folder is /luma , settings are right (splash before payloads), but I see no splash screen at all. I am using Google Chrome 64bit browser (ver 57), with uBlock Origin activated, and I generated the splash.bin via this:
https://sebascontre.github.io/bootscreen3ds/
With these settings:
[Imgur](http://i.imgur.com/T3sBXqi.png)
But it doesn't work at all, I have had no problem before!
